scripts/api/setup-all.sh

```diff
@@ -27,9 +34,7 @@ done
 command -v jq >/dev/null 2>&1 || { echo >&2 '`jq` ("sed for JSON") is required, but not installed. Download the binary for your platform from http://stedolan.github.io/jq/ and make sure it is in your $PATH (/usr/bin/jq is fine) and executable with `sudo chmod +x /usr/bin/jq`. On Mac, you can install it with `brew install jq` if you use homebrew: http://brew.sh . Aborting.'; exit 1; }
 
 echo "deleting all data from Solr"
-curl http://localhost:8983/solr/collection1/update/json?commit=true -H "Content-type: application/json" -X POST -d "{\"delete\": { \"query\":\"*:*\"}}"
-
-SERVER=http://localhost:8080/api
+curl "${SOLR_URL}/solr/collection1/update/json?commit=true" -H "Content-type: application/json" -X POST -d "{\"delete\": { \"query\":\"*:*\"}}"
```

Comment:
I'd rather remove that line. I know is already part of main and development but IMHO it's quite risky to drop a core, without a warning. Further, I would split this container into one that setups dataverse and another one to setup solr.

Reply:
Actually, this "delete all from Solr" line doesn't run for me:

```
dev_bootstrap> curl: (7) Failed to connect to localhost port 8983 after 0 ms: Couldn't connect to server
```

Perhaps this is because we haven't defined SOLR_URL anywhere? 🤔 

Here's a little more context of what I'm seeing:

```
dev_bootstrap> Now executing bootstrapping script at /scripts/bootstrap/dev/init.sh.
dev_bootstrap> Running base setup-all.sh (INSECURE MODE)...
dev_bootstrap> deleting all data from Solr
dev_bootstrap>   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
dev_bootstrap>                                  Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
dev_bootstrap> curl: (7) Failed to connect to localhost port 8983 after 0 ms: Couldn't connect to server
dev_bootstrap> Setup the metadata blocks
dev_bootstrap>   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
dev_bootstrap>                                  Dload  Upload   Total   Spent    Left  Speed
100    54  100    54    0     0   1031      0 --:--:-- --:--:-- --:--:--  1038
dev_bootstrap>   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
dev_bootstrap>                                  Dload  Upload   Total   Spent    Left  Speed
dev_dataverse> [#|2023-05-25T20:35:23.568+0000|INFO|Payara 5.2022.5|org.jvnet.hk2.osgiadapter|_ThreadID=22;_ThreadName=FelixStartLevel;_TimeMillis=1685046923568;_LevelValue=800;|
dev_dataverse>   Skipping registration of inhabitant for service reference [org.osgi.service.metatype.MetaTypeProvider] as the service object could not be obtained.|#]
dev_dataverse> 
dev_bootstrap> {"status":"OK","data":{"message":"NA value created."}}{"status":"OK","data":{"added":[{"name":"citation","type":"MetadataBlock"}...
```

Reply:
Yeah, as we don't provide that the command fails. But as discussed on Zulip, this is not of much use anyway. Let's just delete this thing.

Reply:
Gone. Deleted. 13f92f0